README.md

```diff
@@ -114,6 +114,13 @@ These files are:
 - Test results in human readable format
 - Test results in XML format (default is xUnit but can be changed via options)
 
+Example for Android apk:
+
+```bash
+xharness android test --output-directory=out --package-name="net.dot.System.Numerics.Vectors.Tests" --app="/path/to/test.apk"
```

Comment:
```suggestion
xharness android test --output-directory=out --package-name=net.dot.System.Numerics.Vectors.Tests --app=/path/to/test.apk
```

Reply:
Oh, sorry, I didn't notice it. Will add in the next commit

Reply:
Don't worry about it much